File: stepik-java-api/src/main/java/org/stepik/api/actions/StepikProgressesAction.java
package org.stepik.api.actions;

import org.jetbrains.annotations.NotNull;
import org.stepik.api.client.StepikApiClient;
import org.stepik.api.queries.progresses.StepikProgressesGetQuery;

/**
 * @author meanmail
 */
public class StepikProgressesAction extends StepikAbstractAction {
    public StepikProgressesAction(@NotNull StepikApiClient stepikApiClient) {
        super(stepikApiClient);
    }

    @NotNull
    public StepikProgressesGetQuery get() {
        return new StepikProgressesGetQuery(this);
    }
}

File: stepik-java-api/src/main/kotlin/org/stepik/api/actions/StepikProgressesAction.kt
package org.stepik.api.actions

import org.stepik.api.client.StepikApiClient
import org.stepik.api.queries.progresses.StepikProgressesGetQuery

class StepikProgressesAction(stepikApiClient: StepikApiClient) : StepikAbstractAction(stepikApiClient) {
    
    fun get(): StepikProgressesGetQuery {
        return StepikProgressesGetQuery(this)
    }
}
